app/poi_api/poi_suggestion.py

```diff
@@ -1,28 +1,66 @@
 from datetime import datetime
+from app.poi_api.errors import POINotFoundError, InvalidPOISuggestionError
+from app.common import BadDataError
+from typing import Dict
+import json
 
 
-class POI_suggestions:
+class POI_suggestion:
     def __init__(
-        self, pid, suggestion_name, notes, submission_time, submitted_by
+        self,
+        pid,
+        suggestion_name,
+        notes,
+        submitted_by,
+        submission_time=datetime.now(),

```

Comment:
```suggestion
        submission_time=datetime.now(timezone.utc),
```
Keep everything utc time zone for simplicity.